dashboards/component/chart/serializers.py

```diff
@@ -191,3 +193,31 @@ class Meta(ModelDataMixin.Meta, PlotlyChartSerializer.Meta):
         pass
 
     _meta: Type["ChartSerializer.Meta"]
+
+class StatSerializer(ModelDataMixin, PlotlyChartSerializer, FilterComponent):
```

Comment:
no need to add this here.  it should stay in the stat folder

Reply:
okay I will fix these and push them with new update